client/blocks/site/index.jsx

```diff
@@ -289,23 +207,9 @@ class Site extends Component {
 								</span>
 							</div>
 						) }
-						{ inlineBadges ? (
-							this.renderDomainAndInlineBadges()
-						) : (
-							<>
-								{ /* eslint-disable wpcalypso/jsx-gridicon-size */ }
-								{ this.renderSiteDomain() }
-								{ this.renderSiteBadges() }
-							</>
-						) }
-
-						{ /* eslint-enable wpcalypso/jsx-gridicon-size */ }
+						{ this.renderSiteDomain() }
+						{ this.renderSiteBadges() }
 					</div>
-					{ this.props.homeLink && this.props.showHomeIcon && (
-						<span className="site__home">
```

Comment:
It looks like this was the only reference to the `site__home` class, and we could remove associated styles?

https://github.com/Automattic/wp-calypso/blob/d9a2d8bb1b36a96eaa2ef5e5e55dea39fc90bec7/client/blocks/site/style.scss#L83-L92

https://github.com/Automattic/wp-calypso/blob/d9a2d8bb1b36a96eaa2ef5e5e55dea39fc90bec7/client/my-sites/sidebar/style.scss#L177-L180